build-tools/xaprepare/xaprepare/ToolRunners/BrewRunner.MacOS.cs

```diff
@@ -21,6 +21,8 @@ public BrewRunner (Context context, Log log = null, string toolPath = null)
 			: base (context, log, toolPath)
 		{
 			ProcessTimeout = TimeSpan.FromMinutes (30);
+			EchoStandardError = true;
```

Comment:
Not sure I see why these changes are needed?  I don' understand the interaction between these properties and the parameters to `Link()`.

Reply:
It's needed because without this we won't **capture** the output and each `xaprepare` run will be marred with the messages below for each dependency check:

```
Warning: Already linked: /usr/local/Cellar/mingw-w64/7.0.0_1
To relink:
  brew unlink mingw-w64 && brew link mingw-w64
```